### DISCUSSION TITLE
Q: 评论不成功，报Failed to fetch

### DISCUSSION THREAD

**Discussion by Winner365-China** [walinejs/waline#2053] [Q&A]
### Describe the bug | 问题描述

我部署Waline，测试评论却一直提示Failed to fetch，如何解决？

### Website URL | 问题网站

https://blog.hicosor.top/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
serverURL 填 `https://admin.blog.hicosor.top` 否则有跨域问题

**Reply by leonardo110** [reply]
但是这样无法管理评论了

---
But I can’t manage comments like this
